package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "keep",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Why do we need this?

Reply:
It was added unintentionally. Will delete this file.